### ISSUE TITLE
Search UI missing bottom input box border in high contrast mode

### ISSUE THREAD

**Issue by mousetraps** [Microsoft/vscode#19313]
See below:
![image](https://cloud.githubusercontent.com/assets/762848/22272961/6c090b80-e252-11e6-814e-21bf45100e4b.png)
